components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -1242,7 +1242,7 @@ spec:
       containers:
       - args:
         - -zap-log-level=6
-        image: quay.io/redhat-appstudio/pipeline-service-exporter:d0109787b251d347f08f5d5e473af7ef5dec54b2
+        image: quay.io/redhat-appstudio/pipeline-service-exporter:baf868c848633117ac2b5ea923ae0e3b646f2330
```

Comment:
this sha is available at https://quay.io/repository/redhat-appstudio/pipeline-service-exporter?tab=tags